app/views/people/landing_pages/_address.html.erb

```diff
@@ -1,36 +1,36 @@
-<%= f.fields_for :addresses, errors: {}, fieldset: true do |address|  %>
-  <%= address.hidden_field :_destroy %>
-  <% kind = address.object.kind %>
-  <% style = "" %>
-  <% div_class = kind+"-div" %>
-  <% if kind != 'home' %>
-    <% style="display:none;" %>
-  <% end %>
-  <div style="<%= style %>" class="row middle-row form-group form-group-lg no-pd address-row <%= div_class %>">
-    <div class="col-md-6 col-sm-6 col-xs-12 form-group form-group-lg no-pd">
-      <%= address.hidden_field :kind %>
-
-      <% if kind == 'home' %>
-        <% required = "required" %>
-      <% end %>
-
-      <%= address.text_field :address_1, placeholder: "ADDRESS LINE 1", class: "floatlabel form-control #{required}" %>
-    </div>
-    <div class="col-md-6 col-sm-6 col-xs-12 form-group form-group-lg no-pd">
-      <%= address.text_field :address_2, placeholder: "ADDRESS LINE 2", class: "floatlabel form-control" %>
-    </div>
-  </div>
-
-
-  <div style="<%= style %>" class="row bottom-row address-row <%= div_class %>">
-    <div class="col-md-4 col-sm-4 col-xs-12 form-group form-group-lg no-pd">
-      <%= address.text_field :city, placeholder: "CITY", class: "floatlabel form-control #{required}" %>
-    </div>
-    <div class="col-md-4 col-sm-4 col-xs-12 form-group form-group-lg no-pd">
-      <%= address.select :state, options_for_select(State::NAME_IDS.map(&:last).sort, selected: address.object.state),  include_blank: "SELECT STATE", class: "address-state capital floatlabel form-control capital address-state #{required}" %>
-    </div>
-    <div class="col-md-4 col-sm-4 col-xs-12 form-group form-group-lg no-pd">
-      <%= address.text_field :zip, placeholder: "ZIP", class: "zip number floatlabel form-control zip #{required}" %>
-    </div>
-  </div>
-<% end%>
+<%= f.fields_for :addresses, errors: {}, fieldset: true do |address|  %>
+  <%= address.hidden_field :_destroy %>
+  <% kind = address.object.kind %>
+  <% style = "" %>
+  <% div_class = kind+"-div" %>
+  <% if kind != 'home' %>
+    <% style="display:none;" %>
```

Comment:
inline style here, but I understand this file isn't new and probably isn't used with bs4 on